packages/host/app/components/operator-mode/create-file-modal.gts

```diff
@@ -765,6 +783,59 @@ export class ${className} extends ${exportName} {
       this.saveError = `Error creating card instance: ${e.message}`;
     }
   });
+
```

Comment:
I make use of create-file-modal bcos other modals assume the existence of a catalog entry. This is not displayed in the top right for simplicity